DisplaySettings/DisplaySettings.cpp

```diff
@@ -1039,9 +1039,14 @@ namespace WPEFramework {
                 if(tvResolutions & dsTV_RESOLUTION_480p)supportedTvResolutions.emplace_back("480p");
                 if(tvResolutions & dsTV_RESOLUTION_576i)supportedTvResolutions.emplace_back("576i");
                 if(tvResolutions & dsTV_RESOLUTION_576p)supportedTvResolutions.emplace_back("576p");
+		if(tvResolutions & dsTV_RESOLUTION_576p50)supportedTvResolutions.emplace_back("720p50");
```

Comment:
this should be 576p50

Reply:
Updated it. That was a mistake on my part, my bad.